### PR TITLE
Player tokens

### PR DESCRIPTION
**Pull request by jsingontiko080**
**All Sprint 3 Changees**
Added:

- Indication for being out of the round and spectating 
- Highlight to display who's turn it currently is
- A counter within each player's view to show how many wins they have
- A round number view at the top of the screen
- A hand view on the left side of the top of the screen to show the excluded card
- A winner view to show the winner of the game as well as their card.

### CODE REVIEW

File: Loot/Game/GameState.swift
Comment (bmichaelr) [inline Loot/Game/GameState.swift:1]: Next Round Response o_o

File: Loot/Game/GameViews/.DS_Store
Comment (bmichaelr) [inline Loot/Game/GameViews/.DS_Store:1]: No DS_STORE allowed

File: Loot/Game/GameViews/SwiftUIView.swift
Comment (bmichaelr) [inline Loot/Game/GameViews/SwiftUIView.swift:1]: What is this used for?

File: Loot/Resources/.DS_Store
Comment (bmichaelr) [inline Loot/Resources/.DS_Store:1]: Another DS_STORE

File: Loot/Resources/Assets.xcassets/.DS_Store
Comment (bmichaelr) [inline Loot/Resources/Assets.xcassets/.DS_Store:1]: Another?!

**Review by bmichaelr — CHANGES_REQUESTED** [review]
Looks good, just get rid of those DS store files and explain that one random file and you are good to go

**Review by jsingontiko080 — COMMENTED** [review]
Thanks for review

**Review by bmichaelr — APPROVED** [review]
Great work!

**Pull request merged**